requests_toolbelt/adapters/ssl_context.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+"""This file contains an implementation of the SSLContextAdapter.
+
+   It requires a version of requests >= 2.4.0.
+"""
+
+import requests
+from requests.adapters import HTTPAdapter
+
+
+class SSLContextAdapter(HTTPAdapter):
+    """An adapter that lets the user inject a custom SSL context for all
+       requests made through it.
+
+       The SSL context will simply be passed through to urllib3, which
+       causes it to skip creation of its own context.
+
+       Note that the SSLContext is not persisted when pickling - this is on
+       purpose.
+       So, after unpickling the SSLContextAdapter will behave like an
+       HTTPAdapter until a new SSLContext is set.
+
+       Example usage:
+
+       .. code-block:: python
+
+           import requests
+           from ssl import create_default_context
+           from requests import Session
+           from requests_toolbelt.adapters.ssl_context import SSLContextAdapter
+
+           s = Session()
+           s.mount('https://', SSLContextAdapter(ssl_context=create_default_context()))
+    """
+
+    def __init__(self, **kwargs):
+        self.ssl_context = None
+        if 'ssl_context' in kwargs:
+            self.ssl_context = kwargs['ssl_context']
+            del kwargs['ssl_context']
+
+        super(SSLContextAdapter, self).__init__(**kwargs)
+
+    def __setstate__(self, state):
+        # SSLContext objects aren't picklable and shouldn't be persisted anyway
+        self.ssl_context = None
+        super(SSLContextAdapter, self).__setstate__(state)
+
+    def init_poolmanager(self, *args, **kwargs):
```

Comment:
Please add docstrings to these methods.

Reply:
Hi @sigmavirus24, 

I'm not sure what kind of information would be expected here in a docstring.

The only useful information I could come up with would be something like "Calls superclass' init_poolmanager method with an extra arg on requests >= 2.4.0". This seems redundant, given that the methods themselves are really not complicated.

Adding docstrings here would also be inconsistent with the implementations of the other Adapters in this module. 

If all methods are supposed to have some kind of docstring despite these arguments, I'll be happy to provide some - just trying to understand the reasoning here.

Regards
Nikos